python/docs/source/conf.py

```diff
@@ -23,7 +23,6 @@
 sys.path.insert(0, os.path.abspath('..'))
 sys.path.append(os.path.abspath('.'))
 sys.path.append(os.path.abspath('..'))
```

Comment:
Probably can remove most of these

Reply:
I'm fine either way, whatever gets this to work

Reply:
Removed.